test-data/unit/check-expressions.test

```diff
@@ -1787,3 +1787,8 @@ reveal_type(a.__pow__(2)) # E: Revealed type is 'builtins.int'
 reveal_type(a.__pow__(a)) # E: Revealed type is 'Any'
 a.__pow__() # E: Too few arguments for "__pow__" of "int"
 [builtins fixtures/ops.pyi]
+
+[case testTypeAnnotationNeededMultipleAssignment]
+x,y = [],[] # E: Need type annotation for 'x' \
```

Comment:
I know this is just a test but why not use PEP 8 rules here? (space after comma)

Reply:
yes yes

Reply:
because I need to read PEP 8 :P